setup.py

```diff
@@ -13,7 +13,7 @@
         'pydash==3.4.3',
         'gevent==1.2.1',
         'python-dateutil==2.6.0',
-        'segment_source>0.9'
+        'segment_source>=0.10.1'
```

Comment:
I know it hasn't been like that before, but let's make it `==0.10.1` to avoid release surprises.